src/App.css

```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css2?family=Balsamiq+Sans:wght@700&display=swap');
```

Comment:
Can we remove fixing formatting for App.css from this PR?

Reply:
@grimxreaper

Reply:
You mean remove the formatting? I can copy paste the previous CSS version, I think that should work because I'm not sure where it was formatted to remove it manually.

Reply:
@grimxreaper You need reset or unstash it from the commit. Make sure you don't have the file open. Your editor automatically formats on save which is why pasting the original text and s  saving won't fix it.

Reply:
@bryzaguy  I did a git reset, and unstaged the CSS file.